strapi/strapi
# [Custom fields] Fix type conversion

### What does it do?

Moves custom field type conversion from the backend to the frontend

### How to test it?

- Add a custom field to any content type or component and click save
- The custom field should save to the shema.json with `type: 'customField'`
